doc/api/test.md

```diff
@@ -1352,8 +1352,15 @@ test('changes a mock behavior once', (t) => {
   const fn = t.mock.fn(addOne);
 
   assert.strictEqual(fn(), 1);
-  fn.mock.mockImplementationOnce(addTwo);
+  assert.strictEqual(fn(), 2);
```

Comment:
In addition to the examples, the docs should probably also have a `* Returns: {MockFunctionContext}` explaining this function returns the context in order to allow chaining

Reply:
thanks, done